src/components/DataGridWithInfiniteScroll/DataGridInfiniteScroll.stories.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import {storiesOf} from "@storybook/react";
+import React from "react";
+import DataGridWithInfiniteScroll, {GridSelectionType,} from "./DataGridWithInfiniteScroll";
+import {DATA, DETAILDATA, issueData} from "./DatagridInfiniteScrollMockData";
+import DataTable from 'react-data-table-component';
+import '../../styles/components/DataGridWithInfinteScroll.scss';
```

Comment:
When we build the Dell Design React Common, the webpack would generate all the styles in main.css. But whereas to export the component, we are building it from tsc, which in turn bundles the components and doesn't bundle the styles. So, we are not importing the style from components. If we are importing within the component, it would look for the styles folder in the build. Since we have all the styles in the main.css. So we have removed the style imports from the components